src/components/Elevation.js

```diff
@@ -15,17 +16,16 @@ const CustomTooltip = ({ active, payload }) => {
 };
 
 const Elevation = ({
-  elevationProfile,
   elevationVisible,
   isMobile,
   mobileView,
   toggleElevationVisibility,
   width,
   height,
 }) => {
-  if (!elevationProfile || !elevationProfile.length) {
-    return <div />;
-  }
+  const elevationProfile = useSelector((state) => state.search.elevationProfile);
+
+  if (!elevationProfile || !elevationProfile.length) return null;
```

Comment:
There is some perhaps overly-coupled UI logic in App.js that I think renders the map at a different height depending on whether or not the Elevation component is rendered. I wonder if this could potentially break down (perhaps momentarily) if App.js thinks that Elevation should be rendered, sizes the map accordingly, but this component returns Null.

I wonder if an empty div is at least capable of being an appropriately sized placeholder in that case.

This could all be theoretical. I recommend testing this interaction

Reply:
The plan is to rip out as much of the mobile-specific logic in JS as possible and instead just rely on CSS flex layouts to handle adjusting component sizes.